src/billable-services/create-edit/add-billable-service.component.tsx

```diff
@@ -150,6 +154,9 @@ const AddBillableService: React.FC<{ editingService?: any; onClose: () => void }
           kind: 'success',
           timeoutInMs: 3000,
         });
+        if (editingService && onServiceUpdated) {
```

Comment:
(suggestion) Since updating the list should happen on both save or edit, we shouldn't need to check if it is in edit mode
```suggestion
        if (onServiceUpdated) {
```

Reply:
@ELVISKATS I see you've resolved this comment without implementing the suggestion or proposing why it shouldn't be. Please don't resolve a comment without taking any action on it